src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/KeyGeneration.cs

```diff
@@ -8,13 +8,13 @@ namespace System.Security.Cryptography.Rsa.Tests
     [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser")]
     public class KeyGeneration
     {
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotAzureLinux))]
```

Comment:
Part of me wants to say "we should make the LegalKeySizes values be correct", but that's easy when it's a compile-time constant change.... harder when it's a "one distro is weird" change.

So... this is fine (cue fire meme).

Reply:
Agree, but I am keeping changes as focused as possible for back porting purposes. I can do that as a follow up.